packages/core/src/lib/implementation/execute-plugin.unit.test.ts

```diff
@@ -118,9 +118,7 @@ describe('executePlugins', () => {
         ] satisfies PluginConfig[],
         { progress: false },
       ),
-    ).rejects.toThrow(
-      'Plugins failed: 1 errors: Audit metadata not found for slug node-version',
-    );
+    ).rejects.toThrow('Audit metadata not found for slug node-version');
```

Comment:
Why shorten the assertion? The `Plugins failed: 1 errors`-prefix is also interesting to check.